### PR TITLE
Feature: M2-838

### PR DESCRIPTION
**Pull request by moiskillnadne**

### CODE REVIEW

File: src/widgets/Header/ui/index.tsx [hunk 1]
```diff
@@ -27,6 +31,8 @@ const Header = (): JSX.Element | null => {
 
   const onLogoClick = () => {
     if (user?.id) {
+      clearSelectedApplet()
+      clearActivity()
```
Comment (BamMironov) [inline src/widgets/Header/ui/index.tsx:35]: It is not clear why we need to perform the clean up before navigating to Applets page

File: src/widgets/LogoutTracker/index.tsx [hunk 2]
```diff
@@ -22,9 +24,10 @@ export const LogoutTracker = ({ children }: LogoutTrackerProps) => {
 
     secureTokensStorage.clearTokens()
     clearUser()
-    clearActivityInProgressState()
+    clearSelectedApplet()
+    clearActivity()
     navigate(ROUTES.login.path)
-  }, [clearUser, logout, navigate, tokens?.accessToken, clearActivityInProgressState])
+  }, [clearActivity, clearSelectedApplet, clearUser, logout, navigate, tokens?.accessToken])
```
Comment (BamMironov) [inline src/widgets/LogoutTracker/index.tsx:30]: Can we just use logout feature here?

**Review by BamMironov — CHANGES_REQUESTED** [review]

File: src/entities/activity/lib/activityGroupBuilder.types.ts [hunk 3]
```diff
@@ -1,11 +1,5 @@
-import { ActivityType } from "~/entities/activity"
-import { ScheduleEvent } from "~/entities/event"
-
-export const enum ActivityPipelineType {
-  NotDefined = 0,
-  Regular,
-  Flow,
-}
+import { ActivityType, ScheduleEvent } from "~/shared/lib"
+import { PipelineType } from "~/shared/utils"
```
[the remark this answers is not in the document]
Comment (moiskillnadne) [reply src/entities/activity/lib/activityGroupBuilder.types.ts:2]: Done

File: src/entities/activity/ui/ActivityCard.tsx [hunk 4]
```diff
@@ -14,8 +15,16 @@ interface ActivityCardProps {
 const ActivityCard = ({ activity, disabled, onActivityCardClick }: ActivityCardProps) => {
   const isDisabled = disabled || activity.status === ActivityStatus.Scheduled
 
+  const { saveActivity } = useActivityState()
+
+  const onHandleClick = () => {
```
[the remark this answers is not in the document]
Comment (moiskillnadne) [reply src/entities/activity/ui/ActivityCard.tsx:20]: Done

File: src/widgets/LogoutTracker/index.tsx (hunk 2 above)
Comment (moiskillnadne) [reply src/widgets/LogoutTracker/index.tsx:30]: Done

**Pull request closed without merging**